### DISCUSSION TITLE
Input.clearAll doesn't clear the (user)actions

### DISCUSSION THREAD

**Discussion by chengenzhao** [AlmasB/FXGL#1234] [Q&A]
Hi:

It seems like there is no way to remove action of input
only add (user)action is permitted?
clear all doesn't work for remove added (user)actions
so is it possible to remove action programmatically?

Thanks

**Comment by AlmasB** [comment]
That is correct. This is not possible by design, all user actions are constructed during early initialisation and can never be removed.
